pkg/security/secl/compiler/generators/accessors/accessors.tmpl

```diff
@@ -416,6 +416,18 @@ func (ev *Event) GetFieldMetadata(field eval.Field) (eval.EventType, reflect.Kin
 }
 
 func (ev *Event) SetFieldValue(field eval.Field, value interface{}) error {
+	sanitizeProcessContext := func() {
```

Comment:
could this be a regular function, defined outside the template, that takes `ev` as an argument ?